demos/motmetrics4norfair/motmetrics4norfair.py

```diff
@@ -62,45 +62,41 @@
 
 accumulator = metrics.Accumulators()
 
+def keypoints_distance(detected_pose, tracked_pose):
+    norm_orders = [1, 2, np.inf]
+    distances = 0
+    diagonal = 0
+
+    hor_min_pt = min(detected_pose.points[:, 0])
+    hor_max_pt = max(detected_pose.points[:, 0])
+    ver_min_pt = min(detected_pose.points[:, 1])
+    ver_max_pt = max(detected_pose.points[:, 1])
+
+    # Set keypoint_dist_threshold based on object size, and calculate
+    # distance between detections and tracker estimations
+    for p in norm_orders:
+        distances += np.linalg.norm(
+            detected_pose.points - tracked_pose.estimate, ord=p, axis=1
+        )
+        diagonal += np.linalg.norm(
+            [hor_max_pt - hor_min_pt, ver_max_pt - ver_min_pt], ord=p
+        )
+
+    distances = distances / len(norm_orders)
+
+    keypoint_dist_threshold = diagonal / 18
```

Comment:
Move the `18` into its own descriptive variable and move it to the top with the other hyperparameters.

Reply:
No problem